src/components/design_system/stepper/StepperCustom.tsx

```diff
@@ -0,0 +1,114 @@
+import { SxProps, Theme } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import Stepper from "@mui/material/Stepper";
+import Step from "@mui/material/Step";
+import StepLabel from "@mui/material/StepLabel";
+import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+import TripOriginRoundedIcon from "@mui/icons-material/TripOriginRounded";
+
+const steps = ["Completed", "Active", "Disabled"];
+
+/*
+// TODO: cross-reference colors below w/`theme.ts` and update both that file and references herein accordingly
+
+colors
+
+not started
+text gray 20 #2A333C
+icon gray 50 #A2ACB3
+line gray 50 #A2ACB3
+
+in progress
+text gray 20 #2A333C
+icon primary 20 #9B93F1
+line primary 20 #9B93F1
+
+complete
+text primary 50 #3023B8
+icon primary 50 #3023B8
+line primary 50 #3023B8
+*/
+const currentStep = 1;
+
+export default function HorizontalLinearAlternativeLabelStepper() {
+  return (
+    <Box sx={{ width: "100%" }}>
+      <Stepper activeStep={currentStep} alternativeLabel connector={null}>
+        {steps.map((label, index) => (
+          <Step
+            key={label}
+            sx={{
+              padding: 0,
+              marginRight: index < steps.length - 1 ? "16px" : 0,
+              "&.Mui-disabled": {
+                //TODO: broken b/c MUI does not add this className as described in MUI docs
```

Comment:
I was under the impression that most of the work you were doing was in changing the MUI theme. It seems like this is duplicating some of that code in the theme. Do you need this component? Or is this better than the changes to the theme? Either way, check to make sure you are not duplicating anything.

Reply:
good catch, this is the original code; I'll clean it up a bit and I'd like to get it checked in and then deleted in a follow-up in case there's a need to return to it partially

Reply:
realized it's already in commit history, so will remove the files